workspace/README.md

```diff
@@ -0,0 +1,93 @@
+# Buf workspace example
+
+> You need to have the [`buf` CLI][install] installed to follow along with this example.
+
+This directory provides an example of a Buf [workspace] that contains multiple Buf [modules] that are meant to be consumed as dependencies by other Protobuf projects. There are two modules in this workspace:
+
+* [`observabilityapi`](./observabilityapi) defines an observability-related service.
+* [`observabilitytypes`](./observabilitytypes) defines many of the types used by the service in the `observabilityapi` module.
+
+## Structure
+
+The basic structure of this workspace:
+
+```
+.
+├── buf.gen.yaml
+├── buf.work.yaml
+├── observabilityapi
+│   ├── api
+│   │   ├── v1
+│   │   │   └── api.proto
+│   │   └── v2
+│   │       └── api.proto
+│   └── buf.yaml
+└── observabilitytypes
+    ├── buf.yaml
+    ├── log
+    │   ├── v1
+    │   │   └── log.proto
+    │   ├── v2
+    │   │   └── log.proto
+    │   └── v3
+    │       └── log.proto
+    └── metric
+        └── v1
+            └── metric.proto
+```
+
+The [workspace configuration][buf-work-yaml] is defined in [`buf.work.yaml`](./buf.work.yaml).
+
+## Linting
+
+To [lint] all of the Buf modules in the workspace:
+
+```sh
+buf lint
+```
+
+To lint specific modules:
+
+```sh
+buf lint observabilityapi
+buf lint observabilitytypes
+```
+
+## Breaking change detection
+
+To perform [breaking change detection][breaking] against the previous commit to the `main` branch of this repository:
+
+```sh
+buf breaking \
+--against ../.git#branch=main,ref=HEAD~1,subdir=workspace
```

Comment:
I understand that this will work, but I had a head scratch moment for understanding why this did `../.git`. I'm not sure if we do this in other places in this repo, but this feels very non copy pastable into someone's regular workflow (not one repo with many buf projects).

Maybe this deserves a callout of why the `../` and `subdir=workspace` are there and how they would be different in the average project?

Reply:
Is there a more idiomatic way to do this? I learned this from the GitHub Actions TBH.

Reply:
I don't think there's a more idiomatic way to do this. It is a little bit unidiomatic to have many separate buf projects/workspaces in one repository, but that's probably the best for buf-examples.

Reply:
Yeah, makes sense. I added a clarifying note and an example command that avoids this malarkey.